integ/test_init/test-guide.md

```diff
@@ -0,0 +1,118 @@
+###  Integration Test for Init Process of Fluent Bit on ECS
+
+Following this instruction, you will take the integration test for init process and get the test result
+
+
+
+#### Requirements
+
+1. [install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+2. [set up AWS account credential](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html)
+3. [install AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html)
+
+
+
+#### Deploy test resources
+
+* Create some Fluent Bit config files locally  (you can use any config files that are valid).
+
+* Upload config files to S3 and get ARNs.
+
+* Create a AWS CDK project.
+
+  ```shell
+  mkdir integ_test
+  cd integ_test
+  cdk init app --language python
```

Comment:
I think that we shouldn't be initing the app in the instructions, I think you should do that and then commit that to source control.  And then I think this might remove the need for the step that confused me here: https://github.com/aws/aws-for-fluent-bit/pull/401#discussion_r939089971